dev/reference/system-databases/information-schema.md

```diff
@@ -420,6 +420,236 @@ COLLATION_CONNECTION: utf8_general_ci
 1 row in set (0.00 sec)
 ```
 
+## TIDB\_INDEXES
+
+`TIDB_INDEXES` 表提供了 TiDB 中索引的一些信息。
+
+```sql
+mysql> desc tidb_indexes\G
+*************************** 1. row ***************************
+       Table: TIDB_INDEXES
+Create Table: CREATE TABLE `TIDB_INDEXES` (
+  `TABLE_SCHEMA` varchar(64) DEFAULT NULL,
+  `TABLE_NAME` varchar(64) DEFAULT NULL,
+  `NON_UNIQUE` bigint(21) unsigned DEFAULT NULL,
+  `KEY_NAME` varchar(64) DEFAULT NULL,
+  `SEQ_IN_INDEX` bigint(21) unsigned DEFAULT NULL,
+  `COLUMN_NAME` varchar(64) DEFAULT NULL,
+  `SUB_PART` bigint(21) unsigned DEFAULT NULL,
+  `INDEX_COMMENT` varchar(2048) DEFAULT NULL,
+  `INDEX_ID` bigint(21) unsigned DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
+1 row in set (0.00 sec)
+```
+
+其中 `INDEX_ID` 是 TiDB 为每个索引分配的唯一 ID，可以用来合其他表格中或者 API 中获取到的 `INDEX_ID` 做关联。
+如，我们在 [SLOW_QUERY 表](#SLOW\_QUERY)中得到了某条 SQL 涉及的 `TABLE_ID` 以及 `INDEX_ID`，这时我们可以通过如下的 SQL 来获取索引的信息：
+
+```sql
+select
+   tidb_indexes.*
+from
+   tidb_indexes,
+   tables
+where
+   tidb_indexes.table_schema = tables.table_schema
+   and tidb_indexes.table_name = tidb_indexes.table_name
+   and tables.tidb_table_id = ?
+   and index_id = ?
+```
+
+## TIDB\_HOT\_REGIONS Table
+
+`TIDB_HOT_REGIONS` 表提供了当前 TiKV 中热点 region 的信息。
```

Comment:
```suggestion
`TIDB_HOT_REGIONS` 表提供了当前 TiKV 中热点 Region 的信息。
```

Reply:
"Region" is a special technical concept in our product. Please don't spell it like "region."